hyper_click/js_path_resolver.py

```diff
@@ -61,11 +70,26 @@ def resolve(self):
             if result:
                 return result
 
+        # Lookup paths
         result = self.resolve_in_lookup_paths(self.str_path)
         if result:
             return result
+
+        # Node modules
         return self.resolve_node_modules(self.str_path, self.current_dir)
 
+    def resolve_from_alias (self, alias, alias_source):
+        print('resolve from alias', alias, alias_source)
+        if (self.str_path == alias):
+            result = self.resolve_relative_to_dir(self.str_path, self.currentRoot)
+            if result:
+                return result
+        elif (self.str_path.startswith(alias + '/')):
+            alias_path = alias_source + self.str_path[len(alias):]
```

Comment:
Looks like `path.join` is more robust way for building paths

```suggestion
            alias_path = path.join(alias_source, self.str_path[len(alias):])
```